File: camerakit/src/main/java/com/carterchen247/camerakit/core/UserActionCallback.java
package com.carterchen247.camerakit.core;

/**
 * Created by kazaf on 2018/1/4.
 */

public interface UserActionCallback {

    void onUsingFrontCamera();

    void onUsingBackCamera();

    void onUsingFlashOnMode();

    void onUsingFlashOffMode();

    void onUsingFlashAutoMode();

    void onCaptureButtonShouldLock();

    void onCaptureButtonShouldUnlock();


}

File: camerakit/src/main/java/com/carterchen247/camerakit/core/UserActionCallback.kt
package com.carterchen247.camerakit.core

interface UserActionCallback {
    fun onUsingFrontCamera()
    fun onUsingBackCamera()
    fun onUsingFlashOnMode()
    fun onUsingFlashOffMode()
    fun onUsingFlashAutoMode()
    fun onCaptureButtonShouldLock()
    fun onCaptureButtonShouldUnlock()
}